packages/npm-dependencies-validation/src/utils/npmUtil.ts

```diff
@@ -0,0 +1,26 @@
+import { spawn } from "child_process";
+
+export function getNPM(): string {
+  return /^win/.test(process.platform) ? "npm.cmd" : "npm";
+}
+
+export function spawnCommand(
+  commandArgs: string[],
+  workingDir: string
+): Promise<any> {
```

Comment:
consider replacing `any` with generic `T` so the invoker can supply the expected return type.
also consider making three variants (ls/prune/install) so we will have an exact return type.

- If you make three variants we can abstract away the general `commandArgs`

Reply:
Done.
Why do we need three variants when the only difference is command args ?